tests/staticFilesServer.test.js

```diff
@@ -138,7 +141,7 @@ describe('Static files server', () => {
       });
 
       describe('Other routes', () => {
-        it('should return hint message for routes not starting with /file/', async() => {
+        it('should return hint message for routes not starting with /file/', async () => {
           const response = await axios.get(`${HOST}/file`);
 
           expect(response.status).toBe(200);
```

Comment:
The expectation for the status code here should be 400, not 200, since the server should return a 400 status for routes not starting with '/file/'.